src/globule.c

```diff
@@ -653,3 +653,129 @@ void set_output_fd(int fd) {
 	globule->output_fd = fd;
 	if (fd != -1) pcapfile_write_header(fd);
 }
+
```

Comment:
i don't think globule.c is the right place for this. globule deals with setting global vars. this seems to be a candidate for session.c

Reply:
The functions for sort p1 and p2, it change the pointers of p1 and p2. No free() and strdup() are used. The function accesses the variable 'globule->p1' directly and not through function.

Reply:
well, then make it use setter and getter instead.

Reply:
Pointer usage for sort is faster.
Using set_p1(), set_p2() is more costly because free() and strdup().
Do create new setter functions, without use of free() and strdup()?

Reply:
faster ? give me a break. this function is only called once. i wouldnt even care if it takes a million cylces.

Reply:
This functions can also be for parameter -p. In my reaver, modified version, I use them with the list of default pins generated to attack.